000-vip-init.php

```diff
@@ -139,7 +144,7 @@
 		'schedule'  => 'hourly',
 		'action'    => 'wpcom_vip_run_jetpack_connection_pilot',
 		'callback'  => array( '\Automattic\VIP\Jetpack\Connection_Pilot', 'do_cron' ),
-		'timestamp' => strtotime( sprintf( '+%d minutes', mt_rand( 1, 60 ) ) ),
+		'timestamp' => strtotime( sprintf( '+%d minutes', mt_rand( 1, 30 ) ) ),
```

Comment:
Just noting this change won't have any effect unless https://github.com/Automattic/Cron-Control/pull/177 is live. Generally I think it's not needed so much anymore if we stick with the phased release structure we have planned.

Reply:
If it doesn't work right now because we need to merge that other PR, I reckon it's better to get rid of this line entirely, as it might lead to confusion. As you point out, we should be good to go with the phased release.